recipes/pyccl/build.sh

```diff
@@ -0,0 +1,17 @@
+
+pyver=`${PYTHON} -c "import sys; print('%d.%d.%d' % (sys.version_info[0], sys.version_info[1], sys.version_info[2]))"`
```

Comment:
The env $PY_VER is not enough for your case?

Reply:
you can also do something like
```
py_ver=$($PYTHON --version)
py_ver=${py_ver//Python /}
```

Reply:
cmake has tended to like the full version in all previous builds. I went ahead with your simpler trick from bash, though I don't expect the sys.version_info stuff to break any time soon.